integration/indexes_compat_test.go

```diff
@@ -845,6 +845,15 @@ func TestCreateIndexesCompatUnique(t *testing.T) {
 			insertDoc: bson.D{{"v", "value"}},
 			new:       true,
 		},
+		"NotUniqueIndex": {
+			models: []mongo.IndexModel{
+				{
+					Keys:    bson.D{{"v", 1}},
+					Options: options.Index().SetUnique(false),
```

Comment:
I like this test, yeah!

Reply:
Do you think this is something we want to merge for this release? Or should I close this PR and let you handle it in your PR? Because it will create conflict for you..

Reply:
@chilagrow I think I can add this test and your fixes to my PR, yes. It's almost ready, so I hope we can merge it before the release.

Reply:
Thanks :pray:

Reply:
Thank _you_! :)